src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
Yeah, if you passed a pointer to the `interface_prefs` field and a pointer to the updating function to apply the change, it might work. Is it overkill for the present situation though?